src/main/scala/mesosphere/marathon/api/v2/PodsResource.scala

```diff
@@ -1,22 +1,39 @@
 package mesosphere.marathon.api.v2
 
+import java.net.URI
+import java.nio.charset.StandardCharsets
 import javax.inject.Inject
 import javax.servlet.http.HttpServletRequest
 import javax.ws.rs._
-import javax.ws.rs.core.{ Context, MediaType, Response }
+import javax.ws.rs.core.{Context, MediaType, Response}
 
+import akka.event.EventStream
 import com.codahale.metrics.annotation.Timed
+import com.wix.accord.Validator
+import com.wix.accord.dsl._
 import mesosphere.marathon.MarathonConf
-import mesosphere.marathon.api.{ AuthResource, MarathonMediaType, RestResource }
-import mesosphere.marathon.plugin.auth.{ Authenticator, Authorizer }
+import mesosphere.marathon.api.{AuthResource, MarathonMediaType, RestResource}
+import mesosphere.marathon.core.base.Clock
+import mesosphere.marathon.core.event._
+import mesosphere.marathon.core.pod.PodDefinition
+import mesosphere.marathon.plugin.auth._
+import mesosphere.marathon.raml.{Network, PodDef, PodStatus}
+import spray.json._
 
 @Path("v2/pods")
 @Consumes(Array(MediaType.APPLICATION_JSON))
 @Produces(Array(MarathonMediaType.PREFERRED_APPLICATION_JSON))
 class PodsResource @Inject() (
     val config: MarathonConf,
     val authenticator: Authenticator,
-    val authorizer: Authorizer) extends RestResource with AuthResource {
+    val authorizer: Authorizer)(
+    implicit
```

Comment:
does guice even support this? even if it does, is it worth being inconsistent with AppsResource (what do we really gain)?